.github/workflows/ci.yml

```diff
@@ -54,12 +54,12 @@ jobs:
 
   # Run the C integration tests.
   c-tests:
-    uses: lf-lang/lingua-franca/.github/workflows/c-tests.yml@master
+    uses: lf-lang/lingua-franca/.github/workflows/c-tests.yml@federated-delay-fix
```

Comment:
Wait, we probably don't want these changes in ci.yml to be committed when this is merged to master, right?

Reply:
We do. After committing, it's a matter of changing these to `master`. This will work as long as the referenced branch exists.